src/components/hooks/use-client-information.js

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from 'react'
+import { getClientInformation, getDomain } from 'common/utility'
+
+export const useClientInformation = () => {
+
+    const [client_information, setClientInformation] = useState(false)
+    const [current_client_information, setCurrentClientInformation] = useState(false)
```

Comment:
I think the naming for these are a little misleading since they're actually boolean values. Could we change them to prefix with `is...` instead?

Reply:
They're meant to be initialised with something falsy...empty objects should be better

Reply:
Ah, empty object is a truthy value, so that won't do. You can initialise it with `null` and also set the state to null in your useEffect and try that.